worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java

```diff
@@ -205,6 +257,7 @@ private void removeInactiveUsers() {
         UserBufferInfo userBufferInfo = next.getValue();
         if (currentTimeMillis - userBufferInfo.getTimestamp() >= userInactiveTimeMills) {
           userBufferStatuses.remove(userIdentifier);
+          userCongestionContextMap.remove(userIdentifier);
```

Comment:
this will potentially remove the UserCongestionControlContext which just put into userCongestionContextMap?

Reply:
it will be removed only reach userInactiveTimeMills.
If getUserCongestionContext is called again after it is removed, a new context will be created.